Charter wiki

Xiribidus opened the pull request.

Xiribidus committed:
First Version

Xiribidus committed:
Added header

Xiribidus committed:
Testing sth

Xiribidus committed:
Test 2

Xiribidus committed:
More Fixes

Xiribidus committed:
Test 3

Xiribidus committed:
Even more fixes

Xiribidus committed:
Even even more fixes

Xiribidus committed:
Fixes

Who would've guessed?

Pyrofab commented:
@Arathain I'll let you check this one

Arathain commented:
I'll quickly publish a release fixing wall torches and adding a soul swapper recipe - shouldn't take long, once that's done and the recipe on the wiki is updated, we're good to go

Xiribidus committed:
Added Soul Swapper Recipe

Arathain commented:
@Pyrofab  All good & ready for merge

Xiribidus committed:
Added spaces

Forgot to commit this from a different device, was already wondering why it didnt have an effect ;-;

Xiribidus committed:
Merge branch 'Charter-Wiki' of https://github.com/Xiribidus/Ladysnake.github.io into Charter-Wiki
